dongle_app/style/app.css

```diff
@@ -186,16 +186,42 @@ button:active{
 }
 
 #tool-buttons button {
-  width: auto;
+  width: 4rem;
   height: 4rem;
   line-height: 4rem;
   font-size: 2rem;
-  background: #00539F;
+  background: rgba(255, 255, 255, 0.85);
   float: right;
-  border-radius: 0.5rem;
+  border-radius: 50%;
   margin-right: 1rem;
   padding: 0 1rem;
-  color: #fff;
+  color: #000;
+}
+
+#tool-buttons button#delete {
+  background: url("img/remove.svg") no-repeat scroll center center / 2rem auto rgba(255, 255, 255, 0.85);
+}
+
+#not-on-line {
+  position: absolute;
+  top: 0px;
+  right: 0px;
+  bottom: 0px;
+  left: 0px;
+  background-color: rgba(0, 0, 0, 0.8);
+  color: #FFF;
+  font-size: 2.5rem;
+  z-index: 1000;
+  padding: 7rem;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  transition: transform 0.5s;
+  transform: translateY(-100%);
+}
+
+#not-on-line.open {
```

Comment:
Why dont we use a more general approach? I would use `body.no-connection #not-online`. If you need to change the prompt for any other element or design, the Javascript would remain the same.